### DISCUSSION TITLE
IgnorePlugin 花了太多时间

### DISCUSSION THREAD

**Discussion by chiaweilee** [umijs/umi#9179] [General]
开启 ignoreMomentLocale 后，会加入 IgnorePlugin。在比较大的项目里，占用编译时间比较长。
有没有更轻量的实现方法？

![image](https://user-images.githubusercontent.com/29817353/187619664-6cfe1215-16ed-449a-bebc-10ff5ea1173a.png)

**Comment by sorrycc** [comment]
这里显示的时间不一定是准的，试试「手动不开启这个功能」，是不是也相差 9s？
